packages/common/src/components/TableView/ManageColumnsToolbar.tsx

```diff
@@ -91,8 +102,8 @@ const ManageColumns = ({
   onChange,
   resourceFields,
   defaultColumns,
+  labels = defaultLabels,
```

Comment:
each label should have it's own prop with default value ...

Reply:
@yaacov  @sjd78 
right now we have 7 labels used by this dialog - adding them all will obfuscate remaining labels used by the component. Using a dictionary seems a better choice but if you prefer them one-by-one then I can refactor the code.

Reply:
solved in https://github.com/kubev2v/forklift-console-plugin/pull/403 by splitting the component